src/state/slices/txHistorySlice/txHistorySlice.ts

```diff
@@ -101,6 +102,25 @@ export const initialState: TxHistory = {
 
 const updateOrInsertTx = (txHistory: TxHistory, tx: Tx, accountId: AccountId) => {
   const { txs } = txHistory
+  const isSpam = (() => {
+    const transfers = tx.transfers
+    // Not an NFT, we don't need to filter this
+    if (!transfers.some(transfer => isNft(transfer.assetId))) return false
+    if (
+      transfers.some(
+        transfer =>
+          nftNameBlacklistRegex.test(
+            `${transfer.token?.name ?? ''} ${transfer.token?.symbol ?? ''}`.toLowerCase(),
+          ) || BLACKLISTED_COLLECTION_IDS.includes(transfer.assetId),
+      )
```

Comment:
and here too
https://github.com/shapeshift/web/pull/5309#discussion_r1329332977

Reply:
https://github.com/shapeshift/web/pull/5310/commits/e80a6fd7181c299433d4311aaf4218e4b4a04314